### ISSUE TITLE
MPI cluster mode 

### ISSUE THREAD

**Issue by angerhang** [bibliolytic/MTlearning#3]
I don't know whether this is a feature that should be merged into the master or should stay in the fork I have. The cluster mode constructs the models for all subjects in parallel when one is to use hold-one-out testing approach, the cluster mode decreases the computation time by a factor of `number of subjects` with just a little overhead. 

However this use case is rather specific to the MPI computing infrastructure, and thus it's @bibliolytic 's decision on whether or not we should include this feature. If so, I would just do a pull request, otherwise I will just let the others use my fork.

**Comment by vinay-jayaram** [comment]
As in you use parfor? That's already an option within MT_baseclass, I don't
see how it helps to have two levels of parallelism...

**Comment by angerhang** [comment]
You are right, parfor can already do part of the job, however, 
the current parallelism can only be done in a stand-alone mode without leveraging
the power of computing cluster.

**Comment by vinay-jayaram** [comment]
Ah it uses the htcondor bindings for Matlab?

**Comment by angerhang** [comment]
yeah

**Comment by vinay-jayaram** [comment]
Then I think it doesn't make sense to publicize it outside the lab

**Comment by angerhang** [comment]
ok, then I will just let the others use my fork :D
